### PR TITLE
[UX] Disable "Theme" field and clear selection when "Use theme style" is set to "No" while creating a form

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

This pull request addresses the user experience issue where the "Theme" field remains visible even when the "Use theme style" option is set to "No." By disabling the "Theme" field under these circumstances, we simplify the form creation process, reducing confusion for users who do not intend to use a theme.

The change will streamline the interface, making it clearer and more intuitive, especially for those who may overlook the relevance of the "Theme" field when it isn't applicable. This update benefits all users involved in form creation by enhancing usability and ensuring that only relevant options are displayed

| Before                                 | After
| -------------------------------------- | ---
|  image                                      | image

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Forms page
3. Create a new form
4. Set any theme using the Theme field
5. Then set "Use theme style" to "No." 
6. Verify that the Theme selection is cleared back to "Choose one" and the field itself is now disabled with no click events.

### CODE REVIEW

**Comment by PatrickJenkner** [comment]
I just tested your PR, and the hiding works. However, I might have found another issue:

This is how the form looks when I set the theme to blank, then disable "Use theme style":
image

This is what it looks like if I set the theme to Neopolitan, then disable "Use theme style":
image

So at least some parts of the theme are still being used, and after your PR one can no longer change the theme. I have no idea if this is a different issue or if it is intended that some parts of the theme are still being used. If it is the former, then your PR is fine. If it is the latter, then your PR should be dropped.

**Comment by andersonjeccel** [comment]
@PatrickJenkner Good catch, I'll update the function to set the theme as blank before disabling the option, totally makes sense

**Review by shinde-rahul — APPROVED** [review]
This works as expected.

Thanks @andersonjeccel.

**Review by PatrickJenkner — APPROVED** [review]
Now it works as intended. I also like that the option is now greyed out instead of hidden.
Thanks!

File: app/bundles/FormBundle/Resources/views/Builder/index.html.twig [hunk 1]
```diff
@@ -194,7 +194,7 @@
               {{ form_row(form.noIndex) }}
               {{ form_row(form.inKioskMode) }}
               {{ form_row(form.renderStyle) }}
-              {{ form_row(form.template) }}
+              <div id="theme-selector">{{ form_row(form.template) }}</div>
```
Comment (escopecz) [inline app/bundles/FormBundle/Resources/views/Builder/index.html.twig]: Please add the ID in the form definition itself: https://github.com/mautic/mautic/blob/5.x/app/bundles/FormBundle/Form/Type/FormType.php#L92

Comment (andersonjeccel) [reply app/bundles/FormBundle/Resources/views/Builder/index.html.twig]: Done!

**Comment by andersonjeccel** [comment]
@escopecz Nice catch, I changed the implementation logic and forgot this unused piece of code there
Removed now, we're good to go

**Pull request merged**